fusion/Modules/Lua/cryptomatte_utilities.lua

```diff
@@ -4,11 +4,198 @@ Optional   : cjson
 Created by : Cédric Duriau         [duriau.cedric@live.be]
              Kristof Indeherberge  [xmnr0x23@gmail.com]
              Andrew Hazelden       [andrew@andrewhazelden.com]
-Version    : 1.2.4
+Version    : 1.2.5
 --]]
 
 -- module table
 cryptomatte_utilities = {}
+--bit = require "bit"
+
+-----------------------
+-- bit lib implementions
```

Comment:
Could you link me towards the bit library you tried here?

Reply:
Good question, it's been a while.  The original link was http://luaforge.net/projects/bitlib but the domain is down. And the luaforge github (https://github.com/luaforge) doesn't include bitlib.

There's other repos using it though: https://github.com/q3k/hacklock/blob/master/software/main/lua-libs/bit.lua